### PR TITLE
Led control

### PR DESCRIPTION
**Pull request by Leo-Dan**
now it should be good.

### CODE REVIEW

**Review by saxtot — APPROVED** [review]
Nice and simple!

**Comment by GLDuval** [comment]
Looks good, but there is one small issue that's why a GitHub action is failing. I'll show you how to fix it this week

File: src/renderer/inputSystem/index.ts [hunk 1]
```diff
@@ -52,6 +52,16 @@ const defaultActions: Action[] = [
       })
     },
   },
+
+  {
+    name: 'wristlights',
+    bindings: [{ type: 'gamepadBtnDown', button: buttonMappings.X }],
+    perform: () => {
+      rosClient
+        .callService({ name: '/capra/wrist_light_toggle' })
```
Comment (benmalenfant) [inline src/renderer/inputSystem/index.ts:61]: not a big fan of spliting a line like that, but if it makes software people happy and still works, why not

**Review by benmalenfant — APPROVED** [review]

File: src/renderer/inputSystem/index.ts (hunk 1 above)
Comment (GLDuval) [reply src/renderer/inputSystem/index.ts:61]: That's style rules we have on the UI :)

**Review by GLDuval — APPROVED** [review]

**Pull request merged**